src/components/LootCard/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect, memo } from 'react';
+import { Pressable, Box, HStack, AspectRatio, Heading } from 'native-base';
+import { useNavigation } from '@react-navigation/native';
+import FastImage from 'react-native-fast-image';
+
+import Loading from '@components/Loading';
+import { Game } from '@interfaces/game.dto';
+import rawg from '@services/rawg.api';
+import { AXIS_X_PADDING_CONTENT, RATIO } from '@utils/constants';
+import { GAMEAPI_KEY } from 'react-native-dotenv';
+
+type Props = {
+  id: number;
+};
+
+const LootCard = ({ id }: Props) => {
+  const navigation = useNavigation();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [game, setGame] = useState<Game>({} as Game);
+
+  useEffect(() => {
+    let isMounted = true;
+
+    const getGame = async () => {
+      try {
+        const response = await rawg.get<Game>(`games/${id}?key=${GAMEAPI_KEY}`);
+
+        isMounted && setGame(response.data);
+        isMounted && setIsLoading(false);
+      } catch (err) {
+        console.log('getGame: ', err);
```

Comment:
console.log

Reply:
loot card now has a visual feedback for fetch errors